### PR TITLE
PHPDoc: Fix some param and return types

### PR DESCRIPTION
**Pull request by cconard96**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

### CODE REVIEW

**Review by orthagh — APPROVED** [review]

**Review by trasher — APPROVED** [review]
Well, approved; but I've not reviewed; that would take too much time

**Review by cedric-anne — APPROVED** [review]

File: inc/authldap.class.php [hunk 1]
```diff
@@ -3731,8 +3731,9 @@ static function isLdapPageSizeAvailable($config_ldap, $check_config_value = true
    /**
     * Does LDAP user already exists in the database?
     *
-    * @param string $name User login/name
-    * @param string $sync Sync field
+    * @param string  $name          User login/name
+    * @param integer $authldaps_id  LDAP authentication server ID
+    * @param null    $sync          Sync field
```
Comment (cedric-anne) [inline inc/authldap.class.php:3736]:
```suggestion
    * @param string|null    $sync          Sync field
```

File: inc/computervirtualmachine.class.php [hunk 2]
```diff
@@ -464,7 +464,7 @@ static function getUUIDRestrictCriteria($uuid) {
    /**
     * Find a virtual machine by uuid
     *
-    * @param fields array of virtualmachine fields
+    * @param array fields  Array of virtualmachine fields
```
Comment (cedric-anne) [inline inc/computervirtualmachine.class.php:467]:
```suggestion
    * @param array $fields  Array of virtualmachine fields
```

**Review by cedric-anne — CHANGES_REQUESTED** [review]

File: inc/dbconnection.class.php [hunk 3]
```diff
@@ -491,7 +493,7 @@ function showSystemInformations($width) {
    /**
     * Enable or disable db replication check cron task
     *
-    * @param enable of disable cron task (true by default)
+    * @param boolean Enable or disable cron task (true by default)
```
Comment (cedric-anne) [inline inc/dbconnection.class.php:496]:
```suggestion
    * @param boolean $enable Enable or disable cron task (true by default)
```

File: inc/mailcollector.class.php [hunk 4]
```diff
@@ -1502,7 +1502,7 @@ function getTotalMails() {
     * @param string                     $path     Temporary path
     * @param integer                    $maxsize  Maximum size of document to be retrieved
     * @param string                     $subject  Message ssubject
-    * @param \Laminas\Mail\Storage\Part $part     Message part (for recursive ones)
+    * @param \Laminas\Mail\Storage\Part $subpart  Message part (for recursive ones)
```
Comment (cedric-anne) [inline inc/mailcollector.class.php:1505]:
```suggestion
    * @param string                     $subpart  Subpart index (used in document filenames)
```

**Review by cedric-anne — CHANGES_REQUESTED** [review]

**Comment by cedric-anne** [comment]
Errors I found detected by Scutinizer: https://scrutinizer-ci.com/g/glpi-project/glpi/inspections/f5487ea7-b2db-4f6d-95fa-527becd6b3e5

On this kind of PR, checking Scrutinizer results is usefull.

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**